api/PopStateEvent.json

```diff
@@ -30,10 +30,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": true
+            "version_added": "6"
```

Comment:
This doesn't match `onpopstate` and "popstate_event" data, which is 5/4.2 for Safari/iOS. Can you investigate if this is a case of the `PopStateEvent` interface not being exposed, or perhaps the "popstate" event originally used a plain `Event`?

The updated https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/PopStateEvent should help answer this, but it looks like the harness is broken on Safari 5 now, so that's something to look into as well.

Reply:
I think this is the same problem as in https://github.com/mdn/browser-compat-data/pull/11881#issuecomment-896595350, https://trac.webkit.org/changeset/94493/webkit also added `window.PopStateEvent`.

Reply:
The issues with Safari 5 are actually caused by one of the extensions BrowserStack has loaded, of which appears to conflict with the JSON3 polyfill.  Running Safari 5 on my local VM works no problem.

Using the latest collector (v3.2.9), I tested this event once again in Safari 5.1 and 6, and have found that the `PopStateEvent` was implemented in Safari 6.  As a result, I'm certain that the `onpopstate` handler used a plain event instance before.

Reply:
@vinyldarkscratch can you file an issue about the problem with the JSON3 polyfill? It sounds like we're loading it too early, and loading it when we're going to send the results would be better?

Reply:
Regarding the event by the way, it was a regular event until Safari 6.

Reply:
Great, thanks for confirming! There's already an issue about Safari 5 now so let's goooooo!